node/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@stroeer/tapir",
+  "name": "@stroeer/tapir-v1",
```

Comment:
Das folgende Feld dürfte in der jetzigen Version nicht mehr stimmen:
```json
"files": [
  "v1"
]
```

Wir brauchen dingend einen Test dafür :/

Reply:
Bitte nicht mergen, bevor das gefixt ist.

Reply:
@bobaaaaa what values do we need here? 

```
"files": [
    "article", "section", "core"
  ]
```
?

Reply:
fixed